PIM/Operation/01_ManageProducts.md

```diff
@@ -0,0 +1,313 @@
+   The *Create Offer* window is displayed.
+
+   ![Single connection offer](../../Assets/Screenshots/PIM/Products/List/Offers/SingleConnection.png "[Single connection offer]")
+
+5. Configure the following settings:
+
+  + Click the *Select Connection* drop-down list and select the connection for which you want to create the offer. All active connections are displayed in the list.    
+  The *Select Attribute Set* drop-down list is unlocked.
+
+  + Click the *Select Attribute Set* drop-down list and select the appropriate attribute set for the offer. All active attribute sets are displayed in the list.   
+
+  + Enable the *Generate SKU* toggle to automatically create a SKU for the product offer.   
+
+  + Select the appropriate option for change tracking in the *Change Tracking Mode* section:   
+   ![Radionbutton](../../Assets/Icons/Radiobutton.png "[Radiobutton]") *manual*: Changes in a PIM product must be triggered manually to be applied to the offer. The initial offer is automatically applied.   
+   ![Radionbutton](../../Assets/Icons/Radiobutton.png "[Radiobutton]") *semi-automatic*: Changes in a PIM product are automatically applied to the offer after confirmation in the *Omni-Channel* module. The initial offer is automatically applied without confirmation.   
+   ![Radionbutton](../../Assets/Icons/Radiobutton.png "[Radiobutton]") *semi-automatic, changes must be confirmed by another user*: Changes in a PIM product are automatically applied to the offer after confirmation in the *Omni-Channel* module by another user. The initial offer is only applied after confirmation by another user.    
+   ![Radionbutton](../../Assets/Icons/Radiobutton.png "[Radiobutton]") *automatic*: Changes in a PIM product are automatically applied to the offer. The initial offer is also automatically applied.
+
+  + Click the *Offer Status* drop-down list and select the initial offer status. The statuses below are available:   
+    + **active**: The offer is active. It is displayed in the selected connection and can be sold there.   
+    + **inactive**: The offer is inactive. It is not displayed in the selected connection and cannot be sold there.   
+    + **offline**: The offer is unknown to the selected connection and cannot be sold there.   
+
+
+6. Click the [SAVE] button in the bottom right corner of the *Create Offer* window.   
+The single offer is added to the list of offers. The *Create Offer* window is closed.
+
+  > [Info] The product must be saved to start the creation process of the offer.
+
+
+7. Click the [SAVE] button in the upper right corner.   
+  The *Edit Product* view is closed. The *LIST* tab with the product list is displayed again. The new offer will be created.
+
+  > [Info] It may take some time until the offer has been finally created.
```

Comment:
Vorher "will be created" ist wahrscheinlich aus einem Grund da, aber spricht das positiv übers Produkt? Is es sinnvoll, es so zu formulieren? Oder ist die Erstellung vom Offer von anderen Sachen abhängig?